src/Cake.Common.Tests/Unit/Tools/NUnit/NUnit3RunnerTests.cs

```diff
@@ -218,9 +219,11 @@ public void Should_Set_Commandline_Switches()
                 fixture.Settings.OutputFile = "stdout.txt";
                 fixture.Settings.ErrorOutputFile = "stderr.txt";
                 fixture.Settings.Full = true;
-                fixture.Settings.Results = "NewTestResult.xml";
-                fixture.Settings.ResultFormat = "nunit2";
-                fixture.Settings.ResultTransform = "nunit2.xslt";
+                fixture.Settings.Results = new[]
+                {
+                    new NUnit3Result { FileName = "NewTestResult.xml", Format = "nunit2", Transform = "nunit2.xslt" },
```

Comment:
@ChrisMaddock ok, so it's either `Format` or `Transform`. Makes sense.

Anyhow, we have no sanity check around that currently either https://github.com/cake-build/cake/blob/develop/src/Cake.Common/Tools/NUnit/NUnit3Runner.cs#L128